test/constants.js

```diff
@@ -0,0 +1,85 @@
+const Events = {
```

Comment:
Missing License header.
See the text in existing files.
Should be added to all new files